infra/ansible/roles/plausible_analytics/templates/tournesol-website-analytics.service.j2

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=The Tournesol website analytics
+PartOf=docker.service
```

Comment:
Isn't this `PartOf` sufficient to solve the issue described in https://github.com/tournesol-app/tournesol/issues/1347 ?

According to Systemd documentation
> **PartOf=** Configures dependencies similar to Requires=, but limited to stopping and restarting of units. When systemd stops or restarts the units listed here, the action is propagated to this unit.

Reply:
Good catch, I haven't tried yet.

Reply:
Restarting the docker service automatically restarts tournesol-website-analytics.service :+1: